### DISCUSSION TITLE
Is label smoothing only useful in multi-class recognition?

### DISCUSSION THREAD

**Discussion by irvingzhang0512** [open-mmlab/mmaction2#637] [Q&A]
After reading [related codes](https://github.com/open-mmlab/mmaction2/blob/master/mmaction/models/heads/base.py#L91), it seems label smoothing is only used in multi-class recognition case. 

I think label smoothing is also useful in single-class recognition.

**Comment by irvingzhang0512** [comment]
I guess label smooth is useful in single-class case, but need some extra work(e.g. loss/pipeline) to train models?

**Comment by innerlee** [comment]
Do you have paper references (later than year 2016) about label smoothing in single class classification?

**Comment by irvingzhang0512** [comment]
Actually, my friend gave me this advice(add label smoothing in single-class classification), it works for him in some cases. 

By the way, someone told me that `mix-up` worked in action recognition, too... If `mix-up` works, i'll create a pr

**Comment by irvingzhang0512** [comment]
I don't much about label smooth in action recognition. But for image classification, [this paper](https://arxiv.org/abs/1812.01187) shows that, label smoothing helps to improve ImageNet accuracy.
![image](https://user-images.githubusercontent.com/22089207/101721945-20ab8c00-3ae4-11eb-91c9-8c90879f8f98.png)

**Comment by kennymckormick** [comment]
In my experiments, I found label smoothing doesn't work on TSN series models. For mixup, it seems to work on training 3D models from scratch (details in https://arxiv.org/pdf/2003.13042.pdf). You can conduct experiments to see if label smoothing works for 3D models.
